.circleci/main.yml

```diff
@@ -184,7 +186,7 @@ jobs:
         command: echo "export TEST_DOCKER_HOST=$(ip -4 addr show docker0 | grep -Po 'inet \K[\d.]+')" >> $BASH_ENV
     - run:
         echo TEST_DOCKER_HOST=$TEST_DOCKER_HOST &&
-        make -O -j << pipeline.parameters.make_jobs >> coverage/sharness_tests.coverprofile test/sharness/test-results/sharness.xml TEST_GENERATE_JUNIT=1 CONTINUE_ON_S_FAILURE=1 TEST_DOCKER_HOST=$TEST_DOCKER_HOST
+        make -O -j << pipeline.parameters.make_jobs >> test_sharness_expensive coverage/sharness_tests.coverprofile test/sharness/test-results/sharness.xml CONTINUE_ON_S_FAILURE=1 TEST_DOCKER_HOST=$TEST_DOCKER_HOST
```

Comment:
We were not running expensive tests before. The way the make file is written - https://github.com/ipfs/kubo/blob/master/test/sharness/Rules.mk#L46 - it seems the intention was to execute tests with `TEST_EXPENSIVE=1` though.

Reply:
Ack,  tests flagged as expensive live in `test/sharness/t0040-add-and-cat.sh`:

```console
$ rg EXPENSIVE test/sharness
test/sharness/t0040-add-and-cat.sh
588:  test_expect_success EXPENSIVE "generate 100MB file using go-random" '
592:  test_expect_success EXPENSIVE "sha1 of the file looks ok" '
598:  test_expect_success EXPENSIVE "ipfs add $ADD_FLAGS bigfile succeeds" '
602:  test_expect_success EXPENSIVE "ipfs add bigfile output looks good" '
607:  test_expect_success EXPENSIVE "ipfs cat succeeds" '
611:  test_expect_success EXPENSIVE "ipfs cat output looks good" '
616:  test_expect_success EXPENSIVE "ipfs cat output hashed looks good" '
621:  test_expect_success FUSE,EXPENSIVE "cat ipfs/bigfile succeeds" '
625:  test_expect_success FUSE,EXPENSIVE "cat ipfs/bigfile looks good" '

test/sharness/lib/test-lib.sh
95:test "$TEST_EXPENSIVE" = 1 && test_set_prereq EXPENSIVE
111:  echo '# TEST_EXPENSIVE='"$TEST_EXPENSIVE"

test/sharness/Rules.mk
69:test_sharness_expensive: export TEST_EXPENSIVE=1
```

They seem to add ~5 seconds to the runtime on my PC, I think that is ok. 
But maybe remove the makefile magic and have explicit  `TEST_EXPENSIVE=1` in the workflow, like we do `TEST_NO_PLUGIN`?

Reply:
Done! I replaced `test_sharness_expensive` and `test_sharness_short` directives with a single `test_sharness` one. Now expensive sharness tests are controlled only via `TEST_EXPENSIVE` variable.